aiida_nanotech_empa/parsers/cubegen_pymol_parser.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+"""AiiDA-Gaussian output parser"""
+from __future__ import absolute_import
```

Comment:
I think this isn't needed for Python3, am I wrong?

Reply:
True, i will remove it